<!-- -->
<launch>

  <arg name="rate" default="1.0" />
  <arg name="start" default="50" />
  <arg name="device_name" default="android/galaxy1" />
  <arg name="topic_names_perception" default="/detection/object_tracker/objects" />


  <include file="$(find smartphone_localizer)/launch/phone_localizer.launch">
    <arg name="device_name" default="android" />
    <arg name="ori_fix" value="$(arg ori_fix)" />
    <arg name="phone_init_fix" value="$(arg phone_init_fix)"/>
    <arg name="phone_init_dir" value="$(arg phone_init_dir)"/>
    <arg name="pf_gps_xy_bias" value="$(arg pf_gps_xy_bias)"/>
  </include>

  <include file="$(find smartphone_localizer)/launch/vehicle_phones_manager.launch">
    <arg name="device_name" default="android" />
    <arg name="ori_fix" value="$(arg ori_fix)" />
    <arg name="phone_init_fix" value="$(arg phone_init_fix)"/>
    <arg name="phone_init_dir" value="$(arg phone_init_dir)"/>
    <arg name="topic_names_perception" value="$(arg topic_names_perception)" />
  </include>

  <node type="rqt_console" name="rqt_console" pkg="rqt_console" />
  <node type="rqt_logger_level" name="rqt_logger_level" pkg="rqt_logger_level" />
  <node type="rqt_graph" name="rqt_graph" pkg="rqt_graph" />
  <node type="rviz" name="rviz" pkg="rviz" args="-d $(find smartphone_localizer)/rviz/phone_localizers.rviz" />

</launch>
